### DISCUSSION TITLE
Hire Me (March 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#63293] [Show and tell]
# What is this?

This discussion is for Next.js developers looking for work.

## Candidates

Share your information if you are looking for work.

Please use this format:

> Location:
> Remote:
> Willing to relocate:
> Technologies:
> Resume/CV:
> Website:
> Email:

Good luck with interviewing!

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍

If you're a company looking to hire, please use #63292 instead.

We've had yet another [success story](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript and Mobile Applications  (Check [GitHub](https://github.com/jensen))
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://jensen.social

Previously: EA Sports
Recently shipped: https://alicent.ai/, https://rlife.app

In 2021/2022 I built over **20** web applications which can be found on my [GitHub](https://github.com/jensen) profile.

**I am only looking for contract work at this time.**

**Reply by theonlydaleking** [reply]
Hey mate, Couldn't see an email anywhere. Sent you a LinkedIn message though.

**Comment by superdev87** [comment]
Location: Germany
Remote: Yes
Willing to relocate: No
Technologies:
- Frontend: HTML/CSS/TailwindCSS/MUI/React/Next.js/Angular/Vue/Three.js/Electron
- Backend: Node.js/ExpressJS/Python/Django/Flask/C#/.NET
- DevOps: Git/Gitlab/BitBucket/Docker/Kubernetes

Linkedin: https://linkedin.com/in/alexanderheck429
Email: alexanderheck429@gmail.com

I am looking for **Full Time** / **Contract** work.

**Comment by GetPsyched** [comment]
Location: India
Remote: Yes
Willing to relocate: Yes (within Asia & Europe)
Technologies:
- Languages: TypeScript, Python, Go, Rust, Nix
- Frameworks/Libraries: NextJS, React, Svelte, HTMX, TailwindCSS, NodeJS, Prisma, Drizzle, ExpressJS
- Databases: NoSQL, MySQL, PostgreSQL

Resume/CV: https://getpsyched.dev/resume
LinkedIn: https://www.linkedin.com/in/priyanshutr
Email: `jobs@getpsyched.dev`

**Comment by AnmolDotX** [comment]
> # What is this?
> 
> This discussion is for Next.js developers looking for work.
> 
> ## Candidates
> 
> Share your information if you are looking for work.
> 
> Please use this format:
> 
> > Location:
> > Remote:
> > Willing to relocate:
> > Technologies:
> > Resume/CV:
> > Website:
> > Email:
> 
> Good luck with interviewing!
> 
> ---
> 
> If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍
> 
> If you're a company looking to hire, please use #63292 instead.
> 
> We've had yet another [success story](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!



Location:- India
Remote:- Yes
Willing to Relocate:- Yes, Within India only
Technologies:- React.js, Next.js, TailwindCSS, Node.js, Express.js, MongoDB etc.

Resume/CV :- https://drive.google.com/file/d/11ai_oV3qa1mRkSuwdz-UKfJIJhYRe37S/view?usp=drivesdk

Email:- kumaranmol8611@gmail.com

**Comment by enszrlu** [comment]
Location: UK
Remote: Yes
Willing to relocate: No
Technologies: Full Stack 
Experience: No professional experience, self taught. 
Looking for: Voluntary work or part time role. 
https://github.com/enszrlu

Frontend: HTML/CSS/TailwindCSS/MUI/React/Next.js

Backend: Node.js/ExpressJS/Python/Django/Flask
DevOps: Git/Gitlab/

**Reply by shorthander** [reply]
Hi enszrlu,
please contact me if you are interessted in supporting me building the webapp for our startup. thank you :) Best Regards Adrian

https://www.linkedin.com/in/adrianrinnus/

**Reply by enszrlu** [reply]
I sent you connection request. :)

**Comment by AveshLutchman** [comment]
Location: Trinidad & Tobago
Willing to Relocate: Depends on location
Remote: Yes
Technologies: React/Next.js, Node, Typescript, Tailwind, Svelte, Python, Java/Java Spring, AWS, C, C++

Email: [avesh.lutchman@outlook.com](mailto:avesh.lutchman@outlook.com)
Linkedin: [https://www.linkedin.com/in/avesh-lutchman](https://www.linkedin.com/in/avesh-lutchman)
Portfolio: [https://aveshlutchman.vercel.app/](https://aveshlutchman.vercel.app/)
Resume: [file](https://github.com/vercel/next.js/files/13814827/avesh.lutchman.2.pdf)

**Comment by phaxz10** [comment]
Location: Philippine
Remote: Yes
Willing to relocate: No
Technologies:  Nextjs, Reactjs, Node.js, Python, JS,TS, TailwindCSS
Resume/CV: https://www.linkedin.com/in/jpbautista10/
Website: https://jpbautista.vercel.app
Email: phaxz.bautista10@gmail.com

**Comment by joelcipriano** [comment]
Location: Recife, Brazil.
Remote: Yes.
Willing to relocate: Not at the moment.
Technologies: Angular, React, Next, Turborepo, Postgres, Firebase.
Resume/CV: https://linkedin.com/in/joelcipriano
Website: https://joelcipriano.com.br/
Email: joelcipriano@gmail.com

**Comment by richard-tirta** [comment]
Location: San Francisco, CA, USA.
Remote: Yes
Willing to relocate: No
Technologies: Next.js, React, TypeScript, Javascript, Node.js, Express, Postgres, MySQL, HTML, CSS, SEO.
Resume/CV: https://drive.google.com/file/d/1lDOWYf26GwaQIYJCHmDONCQeKlmM8Hd6/view?usp=sharing
Website: https://richardtirta.com
Email: rt@richardtirta.com

**Comment by menottiRicardo** [comment]
Location: Panama City, Panama
Remote: Yes
Willing to relocate: Yes
Technologies: Next.js, Remix, React, TypeScript, Javascript, Python, AWS, Azure, Node.js.
Website: [portfolio](https://menottin.com/)
Email: [dacorimenotti@gmail.com](emailto:dacorimenotti@gmail.com)

**Comment by nate-vukovich** [comment]
Location: Bleiburg, Austria
Remote: Yes
Willing to relocate: US 100%, other depend
Technologies: NextJS, React, bit of Angular, Astro, GatsbyJS, Node, Express, TRPC, Tanstack Query and the bunch, Tailwind, MUI, bit of AWS, Azure, Docker, bit of PHP, bit of C#, smidget of Rust, MySQL, Postgres, Prisma, Drizzle
Resume/CV: willing to send a PDF :D
Website: https://nvwebd.com
Email: nejc.vukovic@gmail.com or #nejc.vukovic@nvwebd.com

**Comment by JonathanFerreirar** [comment]
Location: Brasil
Remote: Yes
Willing to relocate: Yes

- Technologies: HTML * CSS * TailwindCSS * Shadcn * Radix * React * Next.js * TypeScript * Jest * Storybook * React query
- Resume/CV: https://www.linkedin.com/in/jonathan-rodrigo/
- Website: https://github.com/JonathanFerreirar

Email: jonathanferreirar@gmail.com

**Comment by chrisokwakol** [comment]
Location: Uganda
Remote: Yes
Willing to relocate: No
Technologies: Next.js, Java, WordPress
Resume/CV: https://drive.google.com/file/d/1YJp1at3LKsiXApTRSfLLIcHtX6iZqRje/view?usp=sharing
Website: https://www.linkedin.com/in/chris-okwakol-a55312107/
Email: chrisokwakol@gmail.com

**Comment by sunpietro** [comment]
Location: Poland (can work in US timezones)
Remote: Yes
Willing to relocate: No
Technologies: NextJS, NodeJS, Golang, TypeScript, Potgresql, AWS, CSS, Tailwind
Resume/CV: https://www.linkedin.com/in/piotr-nalepa/
Website: https://blog.piotrnalepa.pl
Email: itcs@piotrnalepa.pl

**Comment by brunubarbosa** [comment]
Location: Brazil
Remote: Yes
Willing to relocate: No
Technologies: ReactJS/RN (5y), Rails (1y)
Resume/CV: https://www.linkedin.com/in/thedevjs/
Email: brunopjsw@gmail.com

**Comment by andentx** [comment]
Location: Vancouver, Washington, USA 
Remote: Yes
 Willing to relocate: No
 
 Technologies: Next.js, TypeScript, Tailwind, Sanity, Shopify

 LinkedIn: https://www.linkedin.com/in/andrewdentdev
 Website: https://andrewdent.dev
GitHub: https://github.com/andentx
 Email: [andrew@andrewdent.dev](mailto:andrew@andrewdent.dev)

**Comment by donkachii** [comment]
Location: Abuja, Nigeria
Remote: Yes 
Willing to relocate: Yes
Technologies: Javascript, reactjs, nextjs, tailwindcss, typescript, nodes, redux, TDD, postgresql, prisma
Resume/CV: https://resume.io/r/jpXQjxqfZ 
Website: https://onyekachinwakaihe.netlify.app/ 
Email: dnwakaihe@gmail.com

**Comment by mustafajeja** [comment]
Location: Cairo, Egypt
Remote: Yes
Willing to relocate: Yes
Technologies: HTML/ CSS, TailwindCSS, Javascript, React, Next JS, Svelte, Laravel, PostgreSQL, MongoDB, Firebase, Graphql
Resume/CV: https://docs.google.com/document/d/1sdCUJizf6jvU5dghtLCdY3yH8LFqk-UW9-xxNpHapPc/edit?usp=sharing
Website:
Email: mustafajeja@gmail.com

**Comment by PrinOrange** [comment]
Location: China
Remote: Yes
Willing to relocate: No. (If you are willing to arrange a work visa for me, then I will be aggreable to relocate.)
Technologies: Next.js, React,Vue, Solid.js, Preact, TailwindCSS, TypeScript, Javascript and other popular frontend technology stacks, I also do Node.js, Nest.js, Express, Postgres, MySQL, and Java, .NET.

Education: Bachelor's degree in software engineering from a university.
Website: https://blog.terminals.run

**Don't worry about my language and technical issues, I can work well in English and cooperate with you to complete your job requirements.**
Email: [prinorange@outlook.com](mailto:prinorange@outlook.com)

**Comment by mino9421** [comment]
Location: Toronto, Ontario, Canada
Remote: Yes
Willing to relocate: North America, Europe, Middle-East
Recent Activities:
```
- Working with Next.JS
- Doing Game Jams using Unity and personal sandbox projects building game systems
```
Technologies:

    - Front End: JavaScript | TypeScript | Node | Vue/Nuxt.js | React/Next.js | Tailwind
    - Back End: Spring Boot (Java) | Laravel (PHP) | ASP.NET MVC Core (C#) | PostgreSQL | MongoDB | Redis
    - State Management Tools: Redux | Pinia | Zustand | Prisma | tRPC | ZOD | GraphQL| Webstock |Socket.io
    - DevOps: AWS |Azure | GCP | Vercel | Docker | K8S | Terraform | Jenkins | Git | Serverless Framework

Resume/CV: https://www.linkedin.com/in/armenlevon/
Website: https://mino9421.github.io/Armen_portfolio/
Email: mino9421@gmail.com

**Comment by modecode22** [comment]
Location: Algeria.
Remote: Yes.
Willing to relocate: Not at the moment.
Technologies: React, Next, Typescript, Golang.
Resume/CV: [cv.pdf](https://drive.google.com/file/d/1K3DZ86vQxTnvPk_K-IvLmuiPnf41WjjK/view?usp=sharing)
Website: https://moncef.net
Email: [aissaoui.moncef@moncef.net](mailto:aissaoui.moncef@moncef.net)

**Comment by tamatamvan** [comment]
Location: Jakarta, Indonesia
Remote: Yes (any timezone)
Willing to relocate: Not at the moment, but open to it in the future
Role: Fullstack Developer (Frontend Heavy)
Technologies: HTML, CSS, Javascript, TypeScript, React, NextJS, Node.js, Express, Nest.js, PostgreSQL, MongoDB, Firebase, Supabase, Tailwind, Laravel, Ruby on Rails, T3 stack, AWS, GCP, Docker, k8s. (Also willing to explore learn any tech stack not mentioned here.)
Website: https://septiantama.id/
CV/Resume: https://septiantama.id/resume
LinkedIn: https://www.linkedin.com/in/septian-tama/
Github: https://github.com/tamatamvan
Email: me@septiantama.id

Hi There, I'm Tama! I'm an experienced fullstack engineer with main expertise in frontend development.
I'm looking for new opportunities and open to full-time / part-time / contract positions.
If you have any opportunities that might be a good fit for me, feel reach me out through my contact above.
Any helps really appreciated, thanks a lot!

**Comment by ajtruex** [comment]
Location: Indianapolis, Indiana
Remote: Yes
Willing to relocate: Possibly
Technologies: Frontend Dev – Nextjs, React, Typescript, Vue.js, Tailwind CSS, Node.js, MySQL, etc. 
Website: [andrewtruex.com](https://andrewtruex.com)
Email: andrewtruex@gmail.com

**Reply by prescottwhite** [reply]
Hey Andrew, just a quick thought on your resume: it isn't clear what you've been doing since 2019 :) I think it would be a good idea to start with what you've been doing lately and have those other jobs be listed after. Maybe also have a sentence or two outlining what kind of role you are seeking.

**Comment by chetanverma16** [comment]
Location: India.
Remote: Yes.
Willing to relocate: Not at the moment.
Technologies: React, Next, Typescript, TailwindCSS, Node.js, Express, Nest.js, TRPC, MySQL, PostgreSQL, Docker, Prisma, Drizzle.
Resume/CV: [cv.pdf](https://chetanvermaa.notion.site/ae429939ecb24b10b76abe5d5207d135)
Website: https://www.chetanverma.com/
Email: [hello@chetanverma.com](mailto:hello@chetanverma.com)

**Comment by emmadal** [comment]
Location: Ivory Coast
Remote: Yes
Willing to relocate: No
Technologies: React.js, GraphQL, Golang, Typescript, Next.js, Docker, Node.js, Express.js, Gin, MongoDB, MySQL, React Native, Firebase, 
Resume/CV: https://linkedin.com/in/edalougou
Website: https://emmadal.vercel.app/
Email: [emmanueldalougou@gmail.com](mailto:emmanueldalougou@gmail.com)

**Comment by karan0805** [comment]
Location: India
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, Node.js, JavaScript, TypeScript, HTML, CSS, Tailwind-CSS, SQL, PostgreSQL, MySQL, Python, Vercel, AWS
Resume/CV: https://karan0805.vercel.app/resume
Website: https://karan0805.vercel.app/
Email: [karangandhi.navsari@gmail.com](mailto:karangandhi.navsari@gmail.com)

**Comment by Itsnotaka** [comment]
Location: Taiwan & Hong Kong (Permanent) / US (F-1)
Remote: Yes
Willing to relocate: Yes
Technologies:JavaScript, TypeScript, React/Next.js, Node.js, tRPC, Tailwind CSS, SQL, Websocket, Electron, Vercel, AWS
Resume/CV: [https://cv.nameisdaniel.com](https://cv.nameisdaniel.com)
Website: [https://nameisdaniel.com](https://nameisdaniel.com)
Email: [nameisdaniel@flowapp.so](mailto:nameisdaniel@flowapp.so) / [daniel.fu90@gmail.com](mailto:daniel.fu90@gmail.com)

**Comment by sagardwivedi** [comment]
---

**Location:** India  
**Remote:** Yes  
**Willing to relocate:** No  

---

**Technologies:**

- **Frontend:** Proficient in HTML, CSS, JavaScript, Next.js, Remix and Tailwind CSS. Experienced in crafting responsive and visually appealing user interfaces. Currently exploring advanced state management with Zustand.
  
- **Backend:** Skilled in Express.js, FastAPI, Postgres, MySQL, and MongoDB. Capable of developing robust backend systems to support dynamic web applications. Utilizing Prisma for efficient database management.
  
- **Mobile Development:** Familiar with mobile development concepts and tools, including React Native and Flutter, with the ability to leverage existing web development skills for mobile app development.

- **Languages:** Fluent in JavaScript (including TypeScript) and Python, with a strong understanding of their respective ecosystems and best practices. Eager to delve deeper into advanced concepts and methodologies.

- **Tools:** Experienced with Git for version control and deployment workflows. Proficient in deploying applications using Vercel and other cloud platforms. Familiar with Docker for containerization and eager to expand knowledge in this area.

---

**Experience:**

- **Personal Projects:** Solely developed and managed personal projects, showcasing expertise in web development technologies and frameworks.

---

**Resume/CV:**  
[View Resume/CV](https://sagardwivedi.vercel.app/)

**Website:**  
[Visit Website](https://sagardwivedi.vercel.app/)

**Email:**  
sagar.dw20@gmail.com

---

**Comment by Phoenix-Alpha** [comment]
Location: Bulgaria(flexible with time zone)
Remote: Yes
Willing to relocate: Probably No.
Technologies:
- Frontend: JS/TS | Vue/Nuxt.js | React/Next.js | Angular 8+ | Slate.js | Tailwind CSS
- Backend: Spring Boot (Java) | Express.js | Nest.js | Django, Flask, FastAPI(python)
- Database: Postgres | MySQL | DynamoDB | Redis | MongoDB
- Other: Redux | Pinia | Jotai | Prisma, TypeORM, Drizzle | GraphQL 
- DevOps: AWS | GCP | Docker | Github Actions

Resume/CV: https://www.betafoenix.net/Resume.pdf
Website: https://www.betafoenix.net/
Email: alexjin0704@gmail.com

**Comment by Jibs-Jibz** [comment]
Location:  Nigeria  
Remote: Yes  
Willing to relocate: Possibly, depending on the opportunity  
Technologies: React, Next.js, Tailwind CSS, Redux, React Query, Next-Auth, JS/TS  
Resume/CV: [View Resume](https://drive.google.com/file/d/15IckOXS4mZc7y3Jr8wkd_lwUYNr3YCyB/view?usp=sharing)  
Website: [https://jibs.vercel.app/](https://jibs.vercel.app/)  
Email: osunkoyajibola@gmail.com

**Comment by nemanjam** [comment]
Location: Serbia, GMT +2
Remote: Yes
Willing to relocate: Maybe
Technologies: React, Next.js, JavaScript/TypeScript, Node.js, TailwindCSS, Jest, Cypress, Prisma, PostrgreSQL, MongoDB, Docker, C#/.NET
Resume/CV: Available on request, please send me an email
GitHub: https://github.com/nemanjam
LinkedIn: https://www.linkedin.com/in/nemanja-mitic
Email: nemanja.mitic.elfak [at] hotmail.com


I am a full stack developer with a Master's degree in Computer Science and 4 years of professional experience with JavaScript and C#/.NET. In the last few years my main focus has been the React/Next.js ecosystem, followed by Node.js. I am a JavaScript professional and a Python beginner interested to get more exposure to it. I have also worked with ASP.NET in the past which is not my primary focus at the moment. I have a basic understanding of DevOps tools.

A quick and easy way to assess my current skill level, the quality of my code and documentation, and my preferred tech stack (Next.js, TypeScript, React Query, Prisma, Postgres, TailwindCSS, Docker, Github Actions) is to check out one of my latest projects: https://github.com/nemanjam/nextjs-prisma-boilerplate .

I have experience working remotely for startups within globally distributed multicultural teams.

I am looking for a full-time remote position. We can discuss relocation after some time spent in the company. You can check out my other projects at https://github.com/nemanjam . I have detailed resume available, you can tell me about your company and project by email: nemanja.mitic.elfak [at] hotmail.com or by LinkedIn: https://www.linkedin.com/in/nemanja-mitic .
